classes/ActionScheduler_AdminView.php

```diff
@@ -28,9 +28,16 @@ public static function instance() {
 	 * @codeCoverageIgnore
 	 */
 	public function init() {
+		$self = self::instance();
 
 		if ( is_admin() && ( ! defined( 'DOING_AJAX' ) || false == DOING_AJAX ) ) {
-			add_filter( 'action_scheduler_post_type_args', array( self::instance(), 'action_scheduler_post_type_args' ) );
+			if ( class_exists( 'WooCommerce' ) ) {
+				add_action( 'woocommerce_admin_status_content_action-scheduler', array( $self, 'render_admin_ui' ) );
+				add_filter( 'woocommerce_admin_status_tabs', array( $self, 'register_menu_woo_tab' ) );
+			} else {
+				add_action( 'admin_menu', array( $self, 'register_menu' ) );
+			}
+			ActionScheduler_ListTable::init();
 		}
 
 		self::$admin_url = admin_url( 'edit.php?post_type=' . ActionScheduler_wpPostStore::POST_TYPE );
```

Comment:
All of the code here responsible for setting up the old admin UI should be deleted when save to do so, or at least deprecated.